tests/scene/test_code_edit.h

```diff
@@ -2314,6 +2314,143 @@ TEST_CASE("[SceneTree][CodeEdit] indent") {
 		}
 	}
 
+	SUBCASE("[CodeEdit] convert indent to tabs") {
+		code_edit->set_indent_size(4);
+		code_edit->set_indent_using_spaces(false);
+
+		// Only line
```

Comment:
```suggestion
		// Only line.
```
Same in comments below.